### PR TITLE
Update crg config for new slurm cluster

### PR DESCRIPTION
**Pull request by JoseEspinosa**
---
Update crg config
---

Please follow these steps before submitting your PR:

- [x] If your PR is a work in progress, include `[WIP]` in its title
- [x] Your PR targets the `master` branch
- [x] You've included links to relevant issues, if any

Steps for adding a new config profile:

- [x] Add your custom config file to the `conf/` directory
- [x] Add your documentation file to the `docs/` directory
- [x] Add your custom profile to the `nfcore_custom.config` file in the top-level directory
- [x] Add your custom profile to the `README.md` file in the top-level directory
- [x] Add your profile name to the `profile:` scope in `.github/workflows/main.yml`

### CODE REVIEW

File: conf/crg.config [hunk 1]
```diff
@@ -5,11 +5,30 @@ params {
     config_profile_url = 'http://www.linux.crg.es/index.php/Main_Page'
 }
 
-process {
-    executor = 'crg'
-    queue    = 'short-centos79,long-centos79'
+def host = System.getenv("HOSTNAME")
+
+if ( host == "genoa64" ) {
```
Comment (emi80) [inline conf/crg.config:10]: Hey @JoseEspinosa, you cannot use `==` here as the host names are something like `genoa64-03`. You need to use [pattern matching](https://www.baeldung.com/groovy-pattern-matching)

**Review by mirpedrol — APPROVED** [review]
LGTM!

**Review by jfy133 — APPROVED** [review]
You don't need to load singularity on the slurm cluster?

Otherwise would be good to document the behaviour on the readme and I think you're good to go :)

File: conf/crg.config [hunk 2]
```diff
@@ -5,11 +5,30 @@ params {
...
+            (task.time <= 3.h   ? '--qos=shorter' :
+            (task.time <= 6.h   ? '--qos=short' :
+            (task.time <= 12.h  ? '--qos=normal' :
+            (task.time <= 24.h  ? '--qos=long' :
+            (task.time <= 48.h  ? '--qos=vlong' :
+            (task.time <= 168.h ? '--qos=marathon' :'--qos=ethernal' )))))) }
+    }
+} else {
+    process {
+        executor = 'crg'
```
Comment (pontus) [inline conf/crg.config:25]: What is this `crg` executor?

File: conf/crg.config [hunk 3]
```diff
@@ -5,11 +5,30 @@ params {
...
-    executor = 'crg'
-    queue    = 'short-centos79,long-centos79'
+def host = System.getenv("HOSTNAME")
+
+if (host ==~ /^genoa64.*/) {
+    process {
+        executor = 'slurm'
+        queue    = 'genoa64'
+        clusterOptions = {
+            task.time <= 1.h    ? '--qos=vshort' :
```
Comment (pontus) [inline conf/crg.config:15]: This is a bit hard to read for me, one might want to consider a switch with a case ranges (see e.g. https://stackoverflow.com/a/66986654) .

**Comment by JoseEspinosa** [comment]
> You don't need to load singularity on the slurm cluster?
> 
> Otherwise would be good to document the behaviour on the readme and I think you're good to go :)

No, at least not currently, will do

File: conf/crg.config (hunk 2 above)
Comment (JoseEspinosa) [reply conf/crg.config:25]: Is a variation of the `sge` executor that @pditommaso implemented for the institute but it is not documented.

File: conf/crg.config [hunk 4]
```diff
@@ -5,11 +5,30 @@ params {
...
+    process {
+        executor = 'crg'
+        beforeScript = 'module load Singularity/4.0.2'
+        queue    = 'short-centos79,long-centos79'
+    }
 }
 
 singularity {
     enabled = true
+    autoMounts = true
```
[the remark this answers is not in the document]
Comment (JoseEspinosa) [reply conf/crg.config:33]: Yes, you are right I will remove it

**Pull request merged**